chef/automate
# Dan/add pg to cfg mgmt

### :nut_and_bolt: Description: What code changed, and why?

Adds a postgres database to the config management service. This will be used to store policyfile change tracking data, but this pull request is just the plumbing:
* create the pg database and user for config management service
* wire up all the habitat things
* Open connection to pg on service startup
* Configure migration tooling

### :chains: Related Resources

#3222

### :athletic_shoe: How to Build and Test the Change

Since this is all plumbing, the app doesn't do anything different, but you can log in to the database and check the migration level:

- Build/start: `build components/automate-deployment/ && build components/config-mgmt-service/ && start_all_services`
- log in w/ psql: `chef-automate dev psql chef_config_mgmt_service`
- show the schema: `\d+`
- show the migration table: `select * from schema_migrations;`
